x-pack/test/performance/tests/playwright/ecommerce_dashboard.ts

```diff
@@ -5,52 +5,69 @@
  * 2.0.
  */
 import Url from 'url';
+import { Page } from 'playwright';
+import { ToolingLog } from '@kbn/dev-utils';
+import { Config } from '@kbn/test';
 import { FtrProviderContext } from '../../ftr_provider_context';
+import { Steps } from '../../services/performance';
 
-export default function ecommerceDashboard({ getService }: FtrProviderContext) {
-  describe('ecommerce_dashboard', () => {
-    const config = getService('config');
-    const performance = getService('performance');
-    const logger = getService('log');
-
-    const { step } = performance.makePage('ecommerce_dashboard');
+function getEcommerceSteps(config: Config, logger: ToolingLog): Steps {
+  return [
```

Comment:
```suggestion
  return <Steps>[
```

Pretty sure this would allow us to remove the `: Steps` return argument declaration and the `: { page: Page }` declarations in each handler.

Side note: breaking the steps out to functions like this creates a lot more type-boilerplate than just defining them inline in the call, like:

```ts
performance.runUserJourney('ecommerce_dashboard', [
  {
    name: '...',
    async handler({ page }) {
    }
  }
])
```